dask_cuda/tests/test_dgx.py

```diff
@@ -10,13 +10,13 @@
 import pytest
 
 from time import sleep
+from dask_cuda.utils import get_gpu_count, get_host_from_cuda_device
+from dask_cuda.initialize import initialize
 from distributed.metrics import time
-from distributed.utils_test import loop  # noqa: F401
+from distributed.utils import get_ip_interface
 from distributed.utils_test import popen
```

Comment:
```suggestion
```

Reply:
I think you meant the line above this.

Reply:
That was really idiotic of me, you indeed meant this line could be removed and instead I removed one that could not. I fixed that now. 🤦